server/proxy/pf_context.c

```diff
@@ -89,6 +93,12 @@ void StaticChannelContext_free(pServerStaticChannelContext* ctx)
 	free(ctx);
 }
 
+static void HashStaticChannelContext_free(void* ptr)
+{
+	pServerStaticChannelContext* ctx = (pServerStaticChannelContext*)ptr;
```

Comment:
Casting void pointers is a superfluous action

```suggestion
	pServerStaticChannelContext* ctx = ptr;
```

Reply:
thx for the hint, will remember that for the next pr ;)